### PR TITLE
Add zh_cn translation and add missing Sugar Water fluid's localization.

### PR DESCRIPTION
**Pull request by mc-kaishixiaxue**

### CODE REVIEW

File: src/main/resources/assets/the_bumblezone/lang/zh_cn.json [hunk 1]
```diff
@@ -1,41 +1,61 @@
...
-	"block.the_bumblezone.empty_honeycomb_brood_block":"空巢脾育苗块",
-	"block.the_bumblezone.honeycomb_brood_block":"巢脾育苗块",
+	"block.the_bumblezone.empty_honeycomb_brood_block":"空蜜脾育幼块",
+	"block.the_bumblezone.honeycomb_brood_block":"蜜脾育幼块",
 	"block.the_bumblezone.sugar_infused_cobblestone":"被糖浸渍的圆石",
 	"block.the_bumblezone.sugar_infused_stone":"被糖浸渍的石头",
-	"block.the_bumblezone.sticky_honey_residue":"粘性蜂蜜残渣",
-	"block.the_bumblezone.sticky_honey_redstone":"粘性蜂蜜红石粉",
+	"block.the_bumblezone.sticky_honey_residue":"黏性蜂蜜残渣",
+	"block.the_bumblezone.sticky_honey_redstone":"黏性蜂蜜红石粉",
```
Comment (TelepathicGrunt) [inline src/main/resources/assets/the_bumblezone/lang/zh_cn.json]: Is this right? When I double check the translation, deepl says "黏性蜂蜜红石粉" is "Viscous honey red stone powder" but it's more of a residue. Not really a powder. For example, let's say I spilled some syrup and it dries out completely. The stuff that is left is an extremely sticky substance attached to the surface. A residue leftover from the syrup. That's is what I was going for. 

Would this be a better translation closer to my original meaning? Or is it incorrect? "黏性蜂蜜红石残留物"

File: src/main/resources/assets/the_bumblezone/lang/zh_cn.json [hunk 2]
```diff
@@ -1,41 +1,61 @@
...
 	"item.the_bumblezone.honey_crystal_shield":"§6蜂蜜结晶盾牌",
 	"item.the_bumblezone.honey_crystal_shield.level_tooltip":"盾牌等级",
 	"item.the_bumblezone.honey_slime_spawn_egg":"蜂蜜史莱姆刷怪蛋",
-	"item.the_bumblezone.music_disc_flight_of_the_bumblebee_rimsky_korsakov": "音乐唱片",
-	"item.the_bumblezone.music_disc_flight_of_the_bumblebee_rimsky_korsakov.desc": "Rimsky Korsakov - Flight of the Bumblebee",
-	"item.the_bumblezone.music_disc_honey_bee_rat_faced_boy": "音乐唱片",
-	"item.the_bumblezone.music_disc_honey_bee_rat_faced_boy.desc": "Rat Faced Boy - Honey Bee",
-	
+	"item.the_bumblezone.music_disc_flight_of_the_bumblebee_rimsky_korsakov":"音乐唱片",
+	"item.the_bumblezone.music_disc_flight_of_the_bumblebee_rimsky_korsakov.desc":"里姆斯基·柯萨科夫 - 野蜂飞舞",
```
Comment (TelepathicGrunt) [inline src/main/resources/assets/the_bumblezone/lang/zh_cn.json]: This is the name of the song and its creator. I believe this is better when not translated. What do you think?

**Review by TelepathicGrunt — CHANGES_REQUESTED** [review]
It looks good! I just have two things that probably should be adjusted before I merge this. The two things are below:

File: src/main/resources/assets/the_bumblezone/lang/zh_cn.json (hunk 1 above)
Comment (mc-kaishixiaxue) [reply src/main/resources/assets/the_bumblezone/lang/zh_cn.json]: changed

File: src/main/resources/assets/the_bumblezone/lang/zh_cn.json (hunk 2 above)
Comment (mc-kaishixiaxue) [reply src/main/resources/assets/the_bumblezone/lang/zh_cn.json]: change revoked

**Comment by TelepathicGrunt** [comment]
Awesome thanks! will merge and it'll be included in bumblezone's next update

Oh yeah, do you wanted to be credited for helping with some translations?

**Comment by mc-kaishixiaxue** [comment]
I don't mind.

**Pull request merged**